healthcare/dicom/system-test/dicom_stores.test.js

```diff
@@ -42,21 +41,16 @@ const dcmFileName = 'IM-0002-0001-JPEG-BASELINE.dcm';
 const resourceFile = `resources/${dcmFileName}`;
 const gcsUri = `${bucketName}/${dcmFileName}`;
 const installDeps = 'npm install';
+let projectId;
 
 // Run npm install on datasets directory because modalities
 // require bootstrapping datasets, and Kokoro needs to know
 // to install dependencies from the datasets directory.
 assert.ok(execSync(installDeps, {cwd: `${cwdDatasets}`, shell: true}));
 
 before(async () => {
-  assert(
-    process.env.GOOGLE_CLOUD_PROJECT,
-    'Must set GOOGLE_CLOUD_PROJECT environment variable!'
-  );
-  assert(
-    process.env.GOOGLE_APPLICATION_CREDENTIALS,
-    'Must set GOOGLE_APPLICATION_CREDENTIALS environment variable!'
-  );
+  projectId = await storage.getProjectId();
```

Comment:
Is this storage.getProjectId() just because the Storage client is being used in this file? As opposed to google.auth.getProjectId().

Reply:
Yup, it calls the same underlying method.  I used `storage.getProjectId()` because we already had a storage client, and it just exposes the underlying auth method.  This method is available on all client libraries.